### ISSUE TITLE
This directory does not appear to be a git repository

### ISSUE THREAD

**Issue by Eladossa** [desktop/desktop#11666]
I want to publish my JS files but I get this message, "_This directory appears to be a Git repository_"
**(This repository doesn't exsit in my github account)**
After I mark "_Add this repository instead_" and trying to _push_   than I get an error that "_The repository does not seem to exist anymore. You may not have access, or it may have been deleted or renamed"_

This is contradictory.

I tried to search a solution for this but nothing seem to help.

Using Github Desktop Version 2.6.3

**Comment by tidy-dev** [comment]
@Eladossa Sorry for your issue. I think I was able to reproduce your scenario via creating a repo with desktop. Publishing it. Then, I deleted the repo from my github.com account. Then, made changes locally, committed them, and cannot push to it any more without receiving the message you are showing.

I was able to correct it by creating a "empty" repo on github.com to match it. Thus if your repo is entitled "super-cool-js-repo" locally, go to your github.com account and [create a repo](https://docs.github.com/en/github/getting-started-with-github/create-a-repo) with the name "super-cool-js-repo" without a readme.  Then, return to desktop and publish again.
